### PR TITLE
Various tweaks to cargo/bounties

### PR DESCRIPTION
**Pull request by GPeckman**
## About The Pull Request
- Bounty pads now benefit from upgraded parts (microlaser and scanning module). When you pick a bounty that you can't complete, you normally have to wait 5 minutes to pick another. Each upgrade will reduce this time by 30 seconds, down to a minimum of 2 minutes.
- Sunglasses can now be bought from cargo. They can be bought as a crate with one pair for 400 credits. Security access is needed to purchase it.
- The gripper gloves single-pack has been changed into the gorilla-gloves single-pack.
- Removed the shady jims bounty.
- Previously, you could only screwdriver/crowbar bounty pads with right click, not left click. Most machines use left click, and the bounty pad does too now.

## Why It's Good For The Game
- This now gives a reason to actually upgrade the bounty pad. It also mitigates situations where you are only offered bounties that are infeasible to complete; This happens a lot to cargo techs, as they can get most if not all bounties offered to them.
- Much like insulated gloves, sunglasses are limited in supply and are desirable for both antags (flash protection) and nonantags (convenient welding protection, revolution protection). This allows sunglasses to be ordered from cargo. Ideally, this will lead to more interaction with cargo.
- The gripper gloves single-pack costs 600 credits. Gripper gloves can already be bought from the sectech vendor, usually for less than that, and the vendor has 5 of them in stock so they almost never run out. There is very little reason to go to the hassle of ordering something from cargo when you could just buy from the sectech instead. By replacing the gripper gloves with gorilla gloves, there is now a reason to order from cargo instead.
- The shady jims bounty has always been a bit of a noob trap, being an item that sounds common but is in reality rare contraband. However, the nerf of smuggler satchel spawns from 10 to 2 (#76621) was the final nail in the coffin. This bounty is just infeasibly hard to complete, and should be removed.
- There is no reason for the bounty pad to use right click instead of left click, and consistency is generally good.

## Changelog
:cl:
add: Bounty pads can now be upgraded, to reduce the time until you can pick a new bounty
add: Sunglasses can now be bought from cargo if you have security access.
balance: Gripper gloves single-pack is now gorilla gloves singlepack
del: Removed shady jims bounty
qol: Bounty pads can now be screwdrivered/crowbarred with left click like every other machine
/:cl:

### CODE REVIEW

**Comment by cnleth** [comment]
I like that sunglasses are being added to cargo, but why sec access for a crate of them? They're not sechuds, you'd expect them to not have sec access restriction

File: code/modules/cargo/packs/security.dm [hunk 1]
```diff
@@ -316,3 +316,11 @@
...
 	contains = list(/obj/item/storage/belt/holster/energy/thermal = 2)
 	crate_name = "thermal pistol crate"
+
+/datum/supply_pack/security/sunglasses
+	name = "Sunglasses Crate"
+	desc = "Three pairs of flash-proof sunglasses."
+	cost = CARGO_CRATE_VALUE * 8
+	access_view = ACCESS_SECURITY
+	contains = list(/obj/item/clothing/glasses/sunglasses = 3)
+	crate_name = "disabler crate"
```
Comment (ExcessiveUseOfCobblestone) [inline code/modules/cargo/packs/security.dm:326]: 😈

**Comment by ExcessiveUseOfCobblestone** [comment]
Why is the goody accessless but add 2 and now you require sec access?

Am I missing something?

**Comment by ExcessiveUseOfCobblestone** [comment]
You should also just update the price, you can pick from a selection so im not entirelY sold why it should be removed

**Comment by atlasle** [comment]
why would you buy a crate of 3 for the price of 4

**Comment by GPeckman** [comment]
> Why is the goody accessless but add 2 and now you require sec access?
> 
> Am I missing something?

I was just doing the same thing insulated gloves did, given that sunglasses and insuls are both scarce and valuable in a similar way. Its a moot point, as after feedback from another maintainer the single-pack has been removed and the crate now has only one pair of sunglasses (with a reduced price, of course).

> You should also just update the price, you can pick from a selection so im not entirelY sold why it should be removed, especially since the pool needs to be as large as possible to work effectively

Even after the removal the assistant pool still has 31 entries left, I don't think the pool being too small is a problem. To put things in perspective, the robotics category has only 5 entries. 

As for why its been removed entirely, its because its so incredibly rare that the bounty just does not work. From code diving, there appears to be one static shady jims packet spawn in Birdshot. Aside from that, the only place they can spawn on-station is from a random contraband spawner, at a 1 in 33 chance. There is one of these static contraband spawners in the North Star contraband locker in the armory. Aside from that, stowaway satchels have 4 contraband items each, and you can order a crate of 5 from cargo for 4000 credits. 

In the average round, there would never be a single shady jims packet spawned anywhere on the station. Having it as a bounty simply does not make sense in light of that.

> why would you buy a crate of 3 for the price of 4

Technically speaking you can't buy the goodies with the departmental budget so that's one reason. But that is a good point WRT to insulated gloves.

**Comment by Dorsisdwarf** [comment]
Why does the crate of 3 sunglasses cost more than buying 3 individual sunglasses?

**Comment by GPeckman** [comment]
> Why does the crate of 3 sunglasses cost more than buying 3 individual sunglasses?

It was originally to mirror costs with insulated gloves, but now it has been changed so that there is only one order of sunglasses (1 pair for 400 credits, sec access required).

File: code/modules/cargo/packs/security.dm (hunk 1 above)
Comment (GPeckman) [reply code/modules/cargo/packs/security.dm]: Fixed

**Comment by GPeckman** [comment]
> I like that sunglasses are being added to cargo, but why sec access for a crate of them? They're not sechuds, you'd expect them to not have sec access restriction

According to another maintainer on discord, there are concerns that easy access to sunglasses could be a revs nerf in disguise. So now they're sec access only.

**Comment by SomeoneYouProbablyKnow** [comment]
Sneaky buff from grippers to gorillas, respectable

**Comment by GPeckman** [comment]
Fixed the linter thing.

> Sneaky buff from grippers to gorillas, respectable

I wouldn't consider it sneaky, the buff and the reasoning for said buff are clearly laid out in the PR.

**Comment by ExcessiveUseOfCobblestone** [comment]
> As for why its been removed entirely, its because its so incredibly rare that the bounty just does not work. From code diving, there appears to be one static shady jims packet spawn in Birdshot. Aside from that, the only place they can spawn on-station is from a random contraband spawner, at a 1 in 33 chance. There is one of these static contraband spawners in the North Star contraband locker in the armory. Aside from that, stowaway satchels have 4 contraband items each, and you can order a crate of 5 from cargo for 4000 credits.

sounds reasonable

File: code/modules/cargo/goodies.dm [hunk 2]
```diff
@@ -93,11 +93,11 @@
 	cost = PAYCHECK_CREW * 8
 	contains = list(/obj/item/clothing/gloves/color/yellow)
 
-/datum/supply_pack/goody/gripper_single
-	name = "Gripper Gloves Single-Pack"
-	desc = "A spare pair of gripper gloves. Perfect for when the security vendor is empty (or when you're not actually a security officer)."
+/datum/supply_pack/goody/gorrilla_single
```
Comment (ArcaneMusic) [inline code/modules/cargo/goodies.dm:96]: gorilla_single*

**Review by ArcaneMusic — CHANGES_REQUESTED** [review]
One minor thing, everything else here seems sensible. Like the upgrade criteria for the bounty pad as well.

**Review by ArcaneMusic — APPROVED** [review]
Looks good to me

**Pull request merged**